src/main/java/codeu/model/data/Tag.java

```diff
@@ -1,25 +1,41 @@
 package codeu.model.data;
 
 import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
 import java.util.UUID;
 
 /** Class representing a tag. */
 public class Tag {
   private final UUID id;
   private final String tag;
   private final Instant creation;
+  private List<Conversation> conversations;
 
   /**
    * Constructs a new Tag.
    *
    * @param id the ID of this Tag
+   * @param conversation The first Conversation this Tag is associated with.
    * @param tag the content of this Tag
    * @param creation the creation time of this Tag
    */
-  public Tag(UUID id, String tag, Instant creation) {
+  public Tag(UUID id, Conversation conversation, String tag, Instant creation) {
     this.id = id;
     this.tag = tag;
     this.creation = creation;
+    this.conversations = new ArrayList<>();
+    this.conversations.add(conversation);
+  }
+
```

Comment:
Is there an advantage with taking in a conversation object for a new tag as opposed to a conversation ID?